cli/ca_mcp.py

```
@@ -0,0 +1,164 @@
+    Query CISO Assistant combined action plan
+    """
+    headers = {
+        "Authorization": f"Token {TOKEN}",
+    }
+    # Get evidence ID by name
+    url = f"{API_URL}/applied-controls/"
+    res = requests.get(url, headers=headers, verify=VERIFY_CERTIFICATE)
+    data = res.json()
+
+    if res.status_code != 200:
+        rprint(f"Error: check credentials or filename.", file=sys.stderr)
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)